src/ui-test/uitest_suite.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the 'License'); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+import { lspExtensionTest } from './tests/lsp_extension_test';
+import { xmlCompletionSupportTest } from './tests/xml_completion_support_test';
+
+describe('Language Support for Apache Camel by Red Hat', function () {
+    lspExtensionTest();
+    xmlCompletionSupportTest();
+});
```

Comment:
better if extra end of line is added at the end of files. it allows better diff on next modifications of the files